vault/dendron.release.2021-11-02.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 4q0nOxVnDvAMD18zyNebk
+title: 'Release Notes (version 0.66)'
+desc: ''
+updated: 1635872354186
+created: 1635870362508
+---
+
+Dendron 0.66 has sprouted  🌱
+
+An early version of task notes is now available to test out! Take a look and feel free to provide feedback on [RFC 13 - Todo Notes in the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358).
+
+This release also includes some breaking changes due to some renaming of commands, which can be referrenced in the changelog.
+
+### Highlights
+- feat(notes): task notes (create modifier & editor highlighting) ([[docs|dendron.topic.tasks]]) (#1583) @kaan
+- feat(workspace): users can convert a local vault to a remote vault, or a remote vault to a local vault with new convert vault command ([[docs|dendron.topic.vaults#vault-convert]]) (#1542) @kaan
+
+### Everything Else
+- enhance(schema): allow untyped templates in schemas ([[docs|dendron.topic.schema#shortened-template-syntax]]) (#1575) @nickolay
+- enhance: add seeds directory to gitignore on workspace creation (#1629) @hikchoi
+- enhance: preview command enablement (#1619) @hikchoi
+- enhance(workspace): add custom icons to dendron tree view (#1635)  @kevin
+- enhance(workspace): Copy Header Reference code action (#1598) @joshi
+- fix: markdown publish to hide block reference anchors (#1577)  @joshi
```

Comment:
add scope for the fix commands. you can see all scopes here -> https://wiki.dendron.so/notes/KhI8VT7HP0dqE3XOx3573.html